src/use_case/login/LoginUserDataAccessInterface.java

```diff
@@ -0,0 +1,11 @@
+package use_case.login;
+
+import entity.User;
+
+public interface LoginUserDataAccessInterface {
+    boolean existsByName(String identifier);
```

Comment:
Change this function signature to `boolean userExists(String username)`

Reply:
This is a great suggestion. I can do the rename in a new PR once this PR is merged.

Reply:
Made the change in this PR.